types/bid.go

```diff
@@ -1,66 +1,84 @@
 package types
 
 import (
+	"encoding/json"
+	"errors"
+
 	"github.com/make-software/casper-go-sdk/types/clvalue"
 	"github.com/make-software/casper-go-sdk/types/key"
 	"github.com/make-software/casper-go-sdk/types/keypair"
 )
 
-// baseBid is an internal structure is using to avoid duplication of code
-type baseBid struct {
+// Bid An entry in the validator map.
+type Bid struct {
```

Comment:
rc3 introduced `minimum_delegation_amount` and `maximum_delegation_amount` fields to this type.
https://github.com/casper-network/casper-sidecar/blob/7438805d58f555404c1a34b4d3280198ca7d5a0d/resources/test/rpc_schema.json#L4926

Reply:
just saw these fields are in the StoredValue PR. Resolving this conversation.